docker/scripts/deprovision.sh

```diff
@@ -199,7 +199,7 @@ if [[ $preserve_data == false ]]; then
 	# LSI MegaRAID and Dell PERC series 9
 	set_autofail_stage "checking/resetting MegaRAID/PERC RAID controllers"
 	# do not do grep -q, it doesn't play well with pipefail when lots of pci devices exist
-	if lspci -nn | grep -v 'SAS3008' | grep LSI >/dev/null && [[ $arch == x86_64 ]]; then
+	if lspci -nn | grep -v 'SAS3008' | grep LSI >/dev/null && [[ $arch == "x86_64" ]]; then
```

Comment:
Do the cheapest check first -- it isn't necessary to run lspci unless arch is x86_64.

Reply:
Nice catch - fixed.